tango/common/util.py

```diff
@@ -153,3 +153,34 @@ def could_be_class_name(name: str) -> bool:
 
 def _is_valid_python_name(name: str) -> bool:
     return bool(name and name[0].isalpha() and name.isalnum())
+
+
+def threaded_generator(g, queue_size: int = 16):
```

Comment:
If you want this in the API docs you can import it in `tango/common/__init__.py`.

Reply:
dcef5b4